reference/explores/macroeconomic_indices.explore.lkml

```diff
@@ -0,0 +1,9 @@
+include: "../views/macroeconomic_indices.view.lkml"
+
+explore: macroeconomic_indices {
```

Comment:
should this explore show up for everyone to see in the explore drop down? 
otherwise you can hide it and people with dev licence can still see it. Depends on the usecase

Reply:
Yeah, I think I want to leave it visible for now